metrics_sdk/lib/opentelemetry/sdk/metrics/export/metric_reader.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        # MetricReader provides a minimal example implementation.
+        # It is not required to subclass this class to provide an implementation
+        # of MetricReader, provided the interface is satisfied.
```

Comment:
I think it's problematic that we have to expose `metric_store` as part of the interface. That should be a SDK-internal concern, which means we probably *do* have to mark it as API-private and require concrete implementations to subclass this class.